cmd/krel/cmd/testdata/changelog-v1.18.0-alpha.3/GetPullRequest-158.json

```diff
@@ -5,7 +5,7 @@
1226 proposes ``mounttest-user`` image to be removed, and ``RunAsUser`` to be used in tests instead. This was proposed here: https://github.com/kubernetes/kubernetes/issues/76342\r\n\r\nThe image used by the Image Promoter (``gcr.io/k8s-staging-test-infra/gcb-docker-gcloud:v20190906-745fed4``) is based on busybox, and thus, the sed binary is actually busybox. ``image-util.sh`` calls ``kube::util::ensure-gnu-sed`` several times, which ensures that a GNU sed binary exists (it checks by greping GNU in its --help output). Obviously, it won't match the busybox sed binary. But the sed usage in ``image-util.sh`` is fairly simple, and the busybox sed is sufficient.\r\n\r\nBumps image versions for: ``jessie-dnsutils``, ``nonewprivs``, ``resource-consumer``, ``sample-apiserver``. These images are included in the ``conformance_images`` that are being built by the Image Promoter, so we're bumping them just to make sure we're not breaking anything and cause all the CIs to fall. We're going to bump the image versions used in tests in a subsequent PR. The image version was not bumped for: ``agnhost``, ``kitten``, ``nautilus``, as they were already bumped by the Image Centralization part 4 PR.\r\n\r\n**Which issue(s) this PR fixes**:\r\n\u003c!--\r\n*Automatically closes linked issue when PR is merged.\r\nUsage: `Fixes #\u003cissue number\u003e`, or `Fixes (paste link of issue)`.\r\n_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*\r\n--\u003e\r\nRelated: #76342\r\nRelated: #81226\r\nRelated: #81170\r\nRelated: #87251\r\n\r\n**Special notes for your reviewer**:\r\n\r\n**Does this PR introduce a user-facing change?**:\r\n\u003c!--\r\nIf no, just write \"NONE\" in the release-note block below.\r\nIf yes, a release note is required:\r\nEnter your extended release note in the block below. If the PR requires additional action from users switching to the new release, include the string \"action required\".\r\n\r\nFor more information on release notes see: https://git.k8s.io/community/contributors/guide/release-notes.md\r\n--\u003e\r\n```release-note\r\nNONE\r\n```\r\n\r\n**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:\r\n\r\n\u003c!--\r\nThis section can be blank if this pull request does not require a release note.\r\n\r\nWhen adding links which point to resources within git repositories, like\r\nKEPs or supporting documentation, please reference a specific commit and avoid\r\nlinking directly to the master branch. This ensures that links reference a\r\nspecific point in time, rather than a document that may change over time.\r\n\r\nSee here for guidance on getting permanent links to files: https://help.github.com/en/articles/getting-permanent-links-to-files\r\n\r\nPlease use the following format for linking documentation:\r\n- [KEP]: \u003clink\u003e\r\n- [Usage]: \u003clink\u003e\r\n- [Other doc]: \u003clink\u003e\r\n--\u003e\r\n```docs\r\n\r\n```\r\n",
```

Comment:
Funny, why did this line change? Besides the newline at the end, I don't see a difference. If the test keeps passing it is not important of course, but its strange isn't it?

Reply:
There was a reference to `k8s-testimages` that my search-replace caught... if this ends up validating against live data I'll revert the change

Reply:
It should not be a problem, this is just replay data for the test so it should be ok :)

Reply:
Ah I see it now:
![image](https://user-images.githubusercontent.com/3935899/134695104-24072783-35ed-474d-bceb-965a2f92f9e9.png)